src/pages/dmt5/_dbanner.js

```diff
@@ -0,0 +1,152 @@
+import React from 'react'
+import styled from 'styled-components'
+import PropTypes from 'prop-types'
+import { Flex } from 'components/containers'
+import { Header, QueryImage } from 'components/elements'
+import { LinkButton } from 'components/form'
+import { localize } from 'components/localization'
+import device from 'themes/device.js'
+
+const Wrapper = styled(Flex)`
+    padding: 0 0 0 12rem;
+    position: relative;
+    height: 40rem;
+    justify-content: flex-start;
+    border-top: 1px solid rgba(151, 151, 151, 0.2);
+    background-color: var(--color-black);
+
+    @media ${device.laptopM} {
+        height: 38.4rem;
+        padding: 0 0 0 8rem;
+    }
+    @media ${device.tabletL} {
+        height: 34.4rem;
+    }
+    @media ${device.tablet} {
+        height: 28.2rem;
+        padding: 0 0 0 4rem;
+    }
+    @media ${device.tabletS} {
+        height: unset;
+        padding: 0 16px 40px 16px;
+    }
+`
+
+const ImageWrapper = styled(Flex)`
+    position: absolute;
+    width: 50rem;
+    top: -2rem;
+    right: 12rem;
+    z-index: 1;
+    height: 100%;
+
+    div {
+        width: 100%;
+    }
+
+    @media ${device.laptopM} {
+        max-width: 43rem;
```

Comment:
lets try to use `px` when using media query 🙏 , as the view will be more consistent